Tasks/KubernetesManifestV0/src/models/TaskInputParameters.ts

```diff
@@ -0,0 +1,6 @@
+"use strict";
+
+export const namespace: string = "namespace";
+export const containers: string = "containers";
+export const canaryPercentage: string = "percentage";
+export const deploymentStrategy: string = "deploymentStrategy";
```

Comment:
Instead of this, you could just get the values of all the parameters? These strings have atmos 1 reference in any flow.